tests/testthat/test-adding.R

```diff
@@ -0,0 +1,7 @@
+test_that("adding() adds two numeric variables together", {
+  x <- 2
+  y <- 3
```

Comment:
@aestears 
I understand this is just testing code but why define x and y when they are never used?